features/environment.py

```diff
@@ -635,11 +635,11 @@ def create_uat_image(context: Context, series: str) -> None:
     ppa = context.config.ppa
     if ppa == "":
         image_name = context.config.cloud_manager.locate_image_name(series)
-        print(
-            "--- Unset UACLIENT_BEHAVE_PPA. Using ubuntu-advantage-tools from image: {}".format(
-                image_name
-            )
-        )
+        msg = (
+            "--- Unset UACLIENT_BEHAVE_PPA. "
+            "Using ubuntu-advantage-tools from image: {}"
+        ).format(image_name)
+        print(msg)
```

Comment:
I also fixed this line-too-long issue but in a slightly different way - so that's why there is a merge conflict here. Your version looks a bit better though so I'd just accept this version.

Reply:
@orndorffgrant I am fine with your fix and I don't think we need an extra commit just to change that. I will drop my changes here.